sdk/communication/azure-communication-chat/azure/communication/chat/_chat_thread_client.py

```diff
@@ -113,7 +113,7 @@ def thread_id(self):
         return self._thread_id
 
     @distributed_trace
-    def update_thread(
+    def update_topic(
         self,
         topic=None,  # type: Optional[str]
```

Comment:
If we are updating the topic-  the `topic` parameter should no longer be optional.